language/German/strings.po

```diff
@@ -4565,1083 +4601,1083 @@ msgstr ""
 msgid "Positional V 9"
-msgstr ""
+msgstr "Lageregler V 9"
 
 #: src/emu/inpttype.ipp:709
 msgctxt "input-name"
 msgid "Positional V 10"
-msgstr ""
+msgstr "Lageregler V 10"
 
 #: src/emu/inpttype.ipp:714
 msgctxt "input-name"
 msgid "Dial"
-msgstr ""
+msgstr "Drehscheibe"
 
 #: src/emu/inpttype.ipp:715
 msgctxt "input-name"
 msgid "Dial 2"
-msgstr ""
+msgstr "Drehscheibe 2"
 
 #: src/emu/inpttype.ipp:716
 msgctxt "input-name"
 msgid "Dial 3"
-msgstr ""
+msgstr "Drehscheibe 3"
 
 #: src/emu/inpttype.ipp:717
 msgctxt "input-name"
 msgid "Dial 4"
-msgstr ""
+msgstr "Drehscheibe 4"
 
 #: src/emu/inpttype.ipp:718
 msgctxt "input-name"
 msgid "Dial 5"
-msgstr ""
+msgstr "Drehscheibe 5"
 
 #: src/emu/inpttype.ipp:719
 msgctxt "input-name"
 msgid "Dial 6"
-msgstr ""
+msgstr "Drehscheibe 6"
 
 #: src/emu/inpttype.ipp:720
 msgctxt "input-name"
 msgid "Dial 7"
-msgstr ""
+msgstr "Drehscheibe 7"
 
 #: src/emu/inpttype.ipp:721
 msgctxt "input-name"
 msgid "Dial 8"
-msgstr ""
+msgstr "Drehscheibe 8"
 
 #: src/emu/inpttype.ipp:722
 msgctxt "input-name"
 msgid "Dial 9"
-msgstr ""
+msgstr "Drehscheibe 9"
 
 #: src/emu/inpttype.ipp:723
 msgctxt "input-name"
 msgid "Dial 10"
-msgstr ""
+msgstr "Drehscheibe 10"
 
 #: src/emu/inpttype.ipp:728
 msgctxt "input-name"
 msgid "Dial V"
-msgstr ""
+msgstr "Drehscheibe V"
 
 #: src/emu/inpttype.ipp:729
 msgctxt "input-name"
 msgid "Dial V 2"
-msgstr ""
+msgstr "Drehscheibe V 2"
 
 #: src/emu/inpttype.ipp:730
 msgctxt "input-name"
 msgid "Dial V 3"
-msgstr ""
+msgstr "Drehscheibe V 3"
 
 #: src/emu/inpttype.ipp:731
 msgctxt "input-name"
 msgid "Dial V 4"
-msgstr ""
+msgstr "Drehscheibe V 4"
 
 #: src/emu/inpttype.ipp:732
 msgctxt "input-name"
 msgid "Dial V 5"
-msgstr ""
+msgstr "Drehscheibe V 5"
 
 #: src/emu/inpttype.ipp:733
 msgctxt "input-name"
 msgid "Dial V 6"
-msgstr ""
+msgstr "Drehscheibe V 6"
 
 #: src/emu/inpttype.ipp:734
 msgctxt "input-name"
 msgid "Dial V 7"
-msgstr ""
+msgstr "Drehscheibe V 7"
 
 #: src/emu/inpttype.ipp:735
 msgctxt "input-name"
 msgid "Dial V 8"
-msgstr ""
+msgstr "Drehscheibe V 8"
 
 #: src/emu/inpttype.ipp:736
 msgctxt "input-name"
 msgid "Dial V 9"
-msgstr ""
+msgstr "Drehscheibe V 9"
 
 #: src/emu/inpttype.ipp:737
 msgctxt "input-name"
 msgid "Dial V 10"
-msgstr ""
+msgstr "Drehscheibe V 10"
 
 #: src/emu/inpttype.ipp:742
 msgctxt "input-name"
 msgid "Track X"
-msgstr ""
+msgstr "Bahn X"
 
 #: src/emu/inpttype.ipp:743
 msgctxt "input-name"
 msgid "Track X 2"
-msgstr ""
+msgstr "Bahn X 2"
 
 #: src/emu/inpttype.ipp:744
 msgctxt "input-name"
 msgid "Track X 3"
-msgstr ""
+msgstr "Bahn X 3"
 
 #: src/emu/inpttype.ipp:745
 msgctxt "input-name"
 msgid "Track X 4"
-msgstr ""
+msgstr "Bahn X 4"
 
 #: src/emu/inpttype.ipp:746
 msgctxt "input-name"
 msgid "Track X 5"
-msgstr ""
+msgstr "Bahn X 5"
 
 #: src/emu/inpttype.ipp:747
 msgctxt "input-name"
 msgid "Track X 6"
-msgstr ""
+msgstr "Bahn X 6"
 
 #: src/emu/inpttype.ipp:748
 msgctxt "input-name"
 msgid "Track X 7"
-msgstr ""
+msgstr "Bahn X 7"
```

Comment:
“Track” here is an abbreviation for “Trackball” – not track as in railway track.

Reply:
D'oh!. Fixed.